frontend/app/model/Faq.scala

```diff
@@ -41,11 +41,11 @@ object Faq {
       Item("What if I just want to give money to the Guardian?", Html("There is also the option to make a <a href=\"https://contribute.theguardian.com/?INTCMP=membership_faq\"> contribution</a> if this is more suitable. All funds raised from readers will go directly into supporting our journalism. We just ask for your name and email address in order for you to do this (we need this in order to process refunds and receipts, and in case of any transactional issues). ")),
       Item("Why don't you just have a paywall?", Html("<p>We are currently running a series of tests asking people who access theguardian.com if they would like to make a financial contribution, either through becoming a Guardian  Supporter or through a one-off contribution. We hope that this will help sustain our journalism and enable us to uncover the stories that need to be told while – crucially – Guardian journalism remains open to all.</p><p>Of course a paywall is something we have looked at and we would never rule it out but putting one up now would get in the way of growing and deepening the relationships that are critical for our future.</p>")),
-      Item("Couldn’t the Guardian just get its owner to provide more money?", Html("The Guardian does not have a wealthy owner pulling the strings. No shareholders, advertisers or billionaire owners can edit our editor. The Scott Trust forms part of a unique ownership structure for the Guardian. Created in 1936, the Trust was set up to secure the financial and editorial independence of the Guardian in perpetuity and to safeguard its journalistic freedom and liberal values, free from commercial or political interference. It reinvests revenue into our journalism, as opposed to into shareholders' pockets. More information can be found <a href=\"https://www.theguardian.com/membership/video/2016/nov/11/who-owns-the-guardian-and-how-is-it-funded\"here</a> and <a href=\"https://www.theguardian.com/the-scott-trust\">here</a>.")),
+      Item("Couldn’t the Guardian just get its owner to provide more money?", Html("The Guardian does not have a wealthy owner pulling the strings. No shareholders, advertisers or billionaire owners can edit our editor. The Scott Trust forms part of a unique ownership structure for the Guardian. Created in 1936, the Trust was set up to secure the financial and editorial independence of the Guardian in perpetuity and to safeguard its journalistic freedom and liberal values, free from commercial or political interference. It reinvests revenue into our journalism, as opposed to into shareholders' pockets. More information can be found <a href=\"https://www.theguardian.com/membership/video/2016/nov/11/who-owns-the-guardian-and-how-is-it-funded\">here</a> and <a href=\"https://www.theguardian.com/the-scott-trust\">here</a>.")),
```

Comment:
This is a hard one to see the difference, but it's to fix a mal-formed link here:
```More information can be found <a href=\"https://www.theguardian.com/membership/video/2016/nov/11/who-owns-the-guardian-and-how-is-it-funded\"**>**here</a>```

Reply:
Thanks, it would have taken me a while to spot that 👍